db/constants.go

```diff
@@ -0,0 +1,5 @@
+package db
+
+const dbEndpoint string = "http://192.168.1.153:8000/"
```

Comment:
We spoke about this in meeting on 23/10 - will need to find a way of avoiding using our IP